### DISCUSSION TITLE
Where can I check the processing progress for GitHub DMCA takedown requests?

### DISCUSSION THREAD

**Discussion by Jiiiiiin** [community/community#60500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, we submitted a DMCA takedown request on the weekend, received an email from GitHub support, and attached a ticket id(Ticket ID: 2242548) to the email, but we don't know where to use this id to query.

Please help us.

**Comment by nirav-gajera** [comment]
To check the processing progress for GitHub DMCA takedown requests, you can follow these steps:

1. Visit the GitHub Support website: https://support.github.com/
2. Scroll down to the bottom of the page and click on the "Contact GitHub" link.
3. On the next page, select the "Submit a request" button.
4. Fill out the required information, including your name, email address, and a brief description of the issue.
5. In the "Subject" field, mention that you are inquiring about the progress of a DMCA takedown request.
6. Provide any additional details or questions related to your request.
7. Submit the form.
By submitting a support request with your inquiry, GitHub Support will be able to provide you with information on the progress of your DMCA takedown request and any updates related to it.

**Reply by Jiiiiiin** [reply]
thx, I found the place you said, and the inquiry ticket has been submitted.
